### PR TITLE
Fix typos COT verification messages.

### PR DESCRIPTION
**Pull request by vedanshujain**
### All Submissions:

-   [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md)?
-   [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
-   [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### How to test the changes in this Pull Request:

1. Run the migration verification process, and check that verification messages change as expected.

### Other information:

-   [x] Have you added an explanation of what your changes do and why you'd like us to include them?
-   [ ] Have you written new tests for your changes, as applicable?
-   [ ] Have you successfully run tests with your changes locally?
-   [x] Have you created a changelog file for each project being changed, ie `pnpm nx changelog <project>`?

### FOR PR REVIEWER ONLY:

-   [ ] I have reviewed that everything is sanitized/escaped appropriately for any SQL or XSS injection possibilities. I made sure Linting is not ignored or disabled.

### CODE REVIEW

**Comment by botwoo** [comment]
:bar_chart: Test reports for this pull request have been published and are accessible through the following links:

- [REST API Test Report](https://woocommerce.github.io/woocommerce-test-reports/pr/33207/api/)
- E2E Test Report (TBC)

Latest commit referenced in the reports: `Minor fixes` d3026db402e8eeba8d91ad370b0f9cfb1a63ed00
This comment will automatically be updated with the latest referenced commit when you push new changes to this pull request.

---

_Visit the [WooCommerce Test Reports](https://woocommerce.github.io/woocommerce-test-reports) homepage to view all published reports. See the [FAQs](https://github.com/woocommerce/woocommerce-test-reports/wiki/FAQs) page if you're having problems accessing them._

File: plugins/woocommerce/changelog/fix-typos [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Fix typos in various cot migration messages.'
```
Comment (jorgeatorres) [inline plugins/woocommerce/changelog/fix-typos]: There's an apostrophe at the end of the line 😅.

File: plugins/woocommerce/src/Database/Migrations/CustomOrderTable/CLIRunner.php [hunk 2]
```diff
@@ -379,8 +384,13 @@ public function verify_cot_data( $args = array(), $assoc_args = array() ) {
 
 			return WP_CLI::error(
 				sprintf(
-					/* Translators: %1$d is the number of migrated orders, %2$f is time taken, %3$d is number of errors and$4%s is formatted array of order ids. */
-					_n( '%1$d order was verified, in %2$f seconds. %3$f error(s) found: %4$s', '%1$d orders were verified in %2$f seconds Please review above errros. %3$d error(s) found %4$s. Please review above errors.', $processed, 'woocommerce' ),
+					/* Translators: %1$d is the number of migrated orders, %2$f is time taken, %3$d is number of errors and $4%s is formatted array of order ids. */
+					_n(
+						'%1$d order was verified in %2$f seconds. %3$f error(s) found: %4$s. Please review above errors.',
+						'%1$d orders were verified in %2$f seconds. %3$d error(s) found %4$s. Please review above errors.',
```
Comment (jorgeatorres) [inline plugins/woocommerce/src/Database/Migrations/CustomOrderTable/CLIRunner.php]:
```suggestion
						'%1$d orders were verified in %2$f seconds. %3$d error(s) found: %4$s. Please review above errors.',
```

File: plugins/woocommerce/src/Database/Migrations/CustomOrderTable/CLIRunner.php [hunk 3]
```diff
@@ -369,7 +369,12 @@ public function verify_cot_data( $args = array(), $assoc_args = array() ) {
 			return WP_CLI::success(
 				sprintf(
 					/* Translators: %1$d is the number of migrated orders and %2$f is time taken */
-					_n( '%1$d order was verified, in %2$f seconds', '%1$d orders were verified in %2$f seconds', $processed, 'woocommerce' ),
+					_n(
+						'%1$d order was verified in %2$f seconds',
```
Comment (jorgeatorres) [inline plugins/woocommerce/src/Database/Migrations/CustomOrderTable/CLIRunner.php]: We probably want to end these sentences with a dot ".", for consistency with the other messages below.

**Review by jorgeatorres — CHANGES_REQUESTED** [review]
Hi @vedanshujain!
Overall this LGTM with only a few minor details that I've pointed out in the feedback. Let me know what you think 🙂.

**Comment by jorgeatorres** [comment]
@vedanshujain: FYI I went ahead and made the relevant changes.

**Review by jorgeatorres — APPROVED** [review]

**Pull request merged**